docs/modules/ROOT/pages/metrics.adoc

```diff
@@ -0,0 +1,1678 @@
+|count
+|the total number of unloaded classes.
+
+|`client.endpoint.count`
+|count
+|the number of active client endpoints for this member
+
+|`client.endpoint.totalRegistrations`
+|count
+|the total number of client endpoint registrations
+
+|`cluster.clock.clusterStartTime`
+|ms
+|the start time of the cluster (when the first node in cluster becomes master, its localClockTime value is saved as clusterStartTime)
+
+|`cluster.clock.clusterTime`
+|ms
+|the elapsed time since the cluster was created
+
+|`cluster.clock.clusterTimeDiff`
+|ms
+|the difference between this member's local clock timestamp and the cluster clock timestamp sent by the master node
+
+|`cluster.clock.clusterUpTime`
+|ms
+|the uptime for the cluster (current time - cluster start time)
+
+|`cluster.clock.localClockTime`
+|ms
+|this member's local clock timestamp
+
+|`cluster.clock.maxClusterTimeDiff`
+|ms
+|the maximum observed cluster time difference
+
+|`cluster.connection.closedTime`
+|count
+|the connection close time for this connection
+
+|`cluster.connection.connectionId`
+|count
+|the connection id for this client connection
+
+|`cluster.connection.eventHandlerCount`
+|count
+|the number of event handler for this client connection
+
+|`cluster.heartbeat.lastHeartbeat`
+|ms
+|the last timestamp of sending heartbeats to the other known cluster members
+
+|`cluster.size`
+|count
+|the size of the cluster
+
+|`cp.atomiclong.value`
+|count
+|the value of this AtomicLong
+
+|`cp.countdownlatch.count`
+|count
+|the initial count of the `ICountDownLatch`
+
+|`cp.countdownlatch.remaining`
+|count
+|the remaining number of expected count-downs
+
+|`cp.countdownlatch.round`
+|count
+|the round number of the `ICountDownLatch`, each time `ICountDownLatch` is initialized with a new count after it downs to zero, a new round begins
+
+|`cp.lock.acquireLimit`
+|count
+|the maximum number of reentrant acquires of this `FencedLock`
+
+|`cp.lock.lockCount`
+|count
+|the number that shows how many times this `FencedLock` acquired
+
+|`cp.lock.owner`
+|count
+|the address of the `FencedLock` owner
+
+|`cp.lock.ownerSessionId`
+|count
+|the sessionId of the `FencedLock` owner
+
+|`cp.semaphore.available`
+|count
+|the number of the remaining available permits
+
+|`cp.semaphore.initialized`
+|count
+|a boolean state, which shows whether semaphore is initialized with a value or not. In exposed metrics it will be 0 when semaphore is not initialized, a positive value otherwise
+
+|`cp.session.creationTime`
+|ms
+|the creation time of this session
+
+|`cp.session.endpoint`
+|
+|the address of the endpoint which session belongs to
+
+|`cp.session.endpointType`
+|
+|the type of the endpoint; either SERVER or CLIENT
+
+|`cp.session.expirationTime`
+|ms
+|the expiration time of the session
+
+|`cp.session.version`
+|count
+|a boolean state, which shows whether semaphore is initialized with a value or not. In exposed metrics it will be 0 when semaphore is not initialized, a positive value otherwise
```

Comment:
Version of the CP session (maybe a clear explanation of what "version" means here).